operations/common-operation.go

```diff
@@ -130,20 +129,18 @@ func setupCerts(insecure bool, certPath, caPath, keyPath string) (*tls.Config, e
 		return nil, nil
 	}
 
-	if certPath == "" || caPath == "" || keyPath == "" {
```

Comment:
this if block should be kept and just `caPath` should be removed and the error message should be changed. Otherwise we have to rely on `LoadX509KeyPair` below to produce a meaningful error message

Reply:
Oh! I missed all the messages until today. Thanks a lot for accepting my small contribution! Do you want me to reintroduce the error message?

Reply:
I already did it myself